components/disposable-focus/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * WordPress dependencies
+ */
+import { Component, findDOMNode } from 'element';
+
+function disposableFocus( WrappedComponent ) {
+	return class extends Component {
+		componentDidMount() {
+			this.activeElement = document.activeElement;
+		}
+
+		componentWillUnmount() {
+			const wrapper = findDOMNode( this );
```

Comment:
I was thinking of providing a config object as a parameter to the HoC as an alternative.

`disposableFocus({ wrapper: 'span' })(Component)`